websocket/go.mod

```diff
@@ -0,0 +1,26 @@
+module github.com/gofiber/contrib/websocket
+
+go 1.20
```

Comment:
Should this be go1.20?

Reply:
it's because of i ran go init on go1.20. Has no affect aganist old go versions unless we run go mod tidy with compat

Reply:
Ahhhh, ok